doc/man1/openssl-cmds.pod

```diff
@@ -2,6 +2,7 @@
 
 =head1 NAME
 
+openssl-cmds,
```

Comment:
Please don't.  This page is a compromise to act as a transfer from old manpage names to new ones.  *not* having this name included is a compromise.

Reply:
okay.